src/Common/src/System/Security/Cryptography/DerEncoder.cs

```diff
@@ -384,6 +385,25 @@ internal static byte[][] SegmentedEncodeOctetString(byte[] data)
             };
         }
 
+        internal static byte[][] SegmentedEncodeNull()
+        {
+            return new byte[][]
```

Comment:
Yeah.  I originally didn't as a defense against becoming public API in the future... but this type is far from being a sensible public API.  So singleton is fine.